#! /usr/bin/env python

PACKAGE='srsnode_joystick_adapter'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("ratio_angular", 
        double_t, 0,
        "Ratio for the linear velocity.", 
        0.6,
        0.1, 1.0
)
gen.add("ratio_linear" ,
        double_t, 0,
        "Ratio for the angular velocity.", 
        0.3,
        0.1, 1.0
)
gen.add("threshold_angular", 
        double_t, 0,
        "Ratio for the linear velocity.", 
        0.06,
        0.0, 1.0
)
gen.add("threshold_linear", 
        double_t, 0,
        "Ratio for the angular velocity.",
        0.3,
        0.1, 1.0
)

exit(gen.generate(PACKAGE, PACKAGE, "Dynamic"))
